app/src/main/res/values/strings.xml

```diff
@@ -90,29 +90,29 @@
+    <string name="prefs_biometriclock_summary">Secure the app with the device\'s biometric lock (e.g. fingerprint, face unlock)</string>
     <string name="prefs_rememberfiltering">Remember sort and filter</string>
     <string name="prefs_rememberfiltering_summary">Saves the chosen sorting and filtering method. Only those marked by a checkbox in the menu are considered.</string>
     <string name="prefs_oldbackups">Filter backups by age</string>
-    <string name="prefs_oldbackups_summary">Filter backups older than value in days. used with the filter \'show old backups\'</string>
+    <string name="prefs_oldbackups_summary">Filter backups older than value in days. Used with the filter \'Show old backups\'</string>
     <string name="prefs_service">Service preferences</string>
     <string name="prefs_encryption">Encryption</string>
     <string name="prefs_encryption_summary">Enable to encrypt data backups</string>
     <string name="prefs_password">Encryption password</string>
     <string name="prefs_password_summary">Set to empty to disable encryption</string>
     <string name="prefs_passwordconfirmation">Encryption password confirmation</string>
     <string name="prefs_password_match_true">The passwords match</string>
-    <string name="prefs_password_match_false">The passwords doesn\'t match</string>
+    <string name="prefs_password_match_false">The passwords don\'t match</string>
     <string name="prefs_excludecache">Exclude app\'s cache</string>
-    <string name="prefs_excludecache_summary">Exclude application\'s cache from data backup</string>
-    <string name="prefs_externaldata">Backup external data</string>
+    <string name="prefs_excludecache_summary">Exclude app\'s cache from data backup</string>
+    <string name="prefs_externaldata">Back up external data</string>
     <string name="prefs_externaldata_summary">Backs up and restores data that is placed on the external storage (e.g. /storage/emulated/0/Android/data/)</string>
-    <string name="prefs_deviceprotecteddata">Backup device protected data</string>
+    <string name="prefs_deviceprotecteddata">Back up device-protected data</string>
     <string name="prefs_deviceprotecteddata_summary">Backs up and restores data that is placed in data/user_de</string>
     <string name="prefs_copyselfapk">Copy own APK</string>
     <string name="prefs_copyselfapk_summary">Place a copy of OAndBackupX in the top level of the backup folder when backed up</string>
     <string name="prefs_acquirewakelock">Prevent sleep</string>
-    <string name="prefs_acquirewakelock_summary">Acquire a wakelock while batch operations are running to prevent sleep. will speed up backups and restores considerably but will consume more energy</string>
+    <string name="prefs_acquirewakelock_summary">Acquire a wakelock while batch operations are running to prevent sleep. Will speed up backups and restores considerably but will consume more energy</string>
```

Comment:
```suggestion
    <string name="prefs_acquirewakelock_summary">Acquire a wakelock while batch operations are running to prevent sleep. Will speed up backups and restorations considerably but will consume more energy</string>
```